Makefile

```diff
@@ -22,6 +22,15 @@
 # `help` target. They look a bit awkward in the variable declarations below
 # since any whitespace added would become part of the variable's default value.
 
+ifeq ($(PKG),)
+ifneq ($(TESTS),)
+$(error TESTS must be specified with PKG (e.g. PKG=./pkg/sql/))
+endif
+ifneq ($(BENCHES),)
+$(error BENCHES must be specified with PKG (e.g. PKG=./pkg/sql/))
+endif
```

Comment:
This isn't strictly true, right? `make test TESTS=TestFoo` will run `TestFoo` in every package in which `TestFoo` exists, which is perhaps desirable. Can we perhaps downgrade these to `$(warning...)`s?

Reply:
Oh, just saw your comment about `PKG = ./pkg/...`. SGTM.